_posts/My Introduction to Wi-Fi 'Hard block' and 'rfkill'.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "My Introduction to Wi-Fi 'Hard block' and 'rfkill'"
+date: 2020-07-26T15:34:30-04:00
+categories:
+  - blog
+  - launch
+tags:
+  - Wi-Fi Hard Block
+  - rfkill
+---
+
+  While I was adjusting the brightness of my laptop, my internet suddenly got disconnected.  I checked router and Wi-Fi connection, they were perfectly fine,  I noticed Wi-Fi symbol disappeared from the drop-down menu, even the settings showed no options to turn it 'on'  and the notification said '**No Wi-Fi Adapter Found**' as shown in the screen shot below : 
+
+
+
+![pic 1](https://user-images.githubusercontent.com/42797168/86515763-07b5f500-be39-11ea-8de8-df6503c4a7f6.png)
+
+Surmising an issue in the Wi-Fi driver, I removed the exisiting Wi-Fi packages using  _'apt'_ & _'purge'_ tools and then re-installed them, but couldn't find any change!!
+
+I then checked the list of pci buses on my system using _'lspci'_ command and  realised  “Network Bus’’ was missing. 
+
+With my collegue’s intervention I came across _‘rfkill’_ tool..
+
+**_'rfkill'_** - is a tool that gives an interface to know the status, activate and deactivate the radio transmittors in a computer.
+It is mainly associated with wireless tramsittors in the system like 'Wi-Fi' , 'Bluetooth' and Mobile Broadband Devices .
+
+When these devices are deactivated they fall into **'soft block'** ( a state in which the device can be re-activated using software) **or**  **'hard block'** ( a state in which the device cannot be re-activated using a software ) 
+
+( References:
```

Comment:
See how references are written in GFM.

Basically you use [[ref1]]  or [[ref any name]] and then add the refs at the end as
```md
Use [ref1] [ref2] [any ref name] in the text, and...

...more text...

...more text with refs...

...rest of the text...

At the end of the document provide the references
[ref1]: https://ayeai.xyz
[ref any name]: https://pvjournal.github.io
```

[ref1]: https://ayeai.xyz
[ref any name]: https://pvjournal.github.io

Reply:
done